tests/nosetests/pyanaconda_tests/module_network_test.py

```diff
@@ -1243,8 +1243,7 @@ def _mock_task_paths(self, task):
         task.NETWORK_SCRIPTS_DIR_PATH = self._mocked_root + "/etc/sysconfig/network-scripts"
         task.PREFIXDEVNAME_DIR_PATH = self._mocked_root + "/etc/systemd/network"
         task.DHCLIENT_FILE_TEMPLATE = self._mocked_root + "/etc/dhcp/dhclient-{}.conf"
-        task.INTERFACE_RENAME_FILE_TEMPLATE = self._mocked_root \
-            + "/etc/systemd/network/10-anaconda-ifname-{}.link"
+        task.SYSTEMD_NETWORK_CONFIG_DIR = self._mocked_root + "/etc/systemd/network"
```

Comment:
Shouldn't this be the constant too?

Reply:
Done in follow-up commit: https://github.com/rhinstaller/anaconda/pull/2626/commits/a5eec328cdd56cb719c9cabbc572d1d82eed78e9